packages/js/tests/decorator/gutenberg.test.js

```diff
@@ -225,6 +227,197 @@ describe( "test getAnnotationsFromBlock", () => {
 			},
 		  ];
 
+		expect( annotations ).toEqual( resultWithAnnotation );
+	} );
+	it( "returns an annotation if there is an applicable marker for the text in a Yoast How-To block", () => {
+		create.mockReturnValue( {
+			// This value will be returned from the last call.
+			// This value matches `jsonDescription` in which inside `getAnnotationsForYoastBlocks` is the last value to be processed.
+			text: "Step by step guide on how to make your cat love you (with or without food bribe).",
+		} )
+			.mockReturnValueOnce( {
+				// This value will be returned from the first call.
+				// This value matches `jsonName` in which inside `getAnnotationsForYoastBlocks` is the first value to be processed.
+				text: "Establish a close contact with your cat",
+			} )
+			.mockReturnValueOnce( {
+				// This value will be returned from the second call.
+				// This value matches `jsonText` in which inside `getAnnotationsForYoastBlocks` is the second value to be processed.
+				text: "According to a research, cats are social animal. Your cat will prefer the hooman than a very tempting snack in a stressful situation.",
+			} );
```

Comment:
`.mockReturnValueOnce()` is used here so that we can have different mocked values for each call of the mocked function. (see [this documentation](https://jestjs.io/docs/mock-function-api#mockfnmockreturnvalueoncevalue) for more detail).